### ISSUE TITLE
freeze_teacher

### ISSUE THREAD

**Issue by 757787182** [nianticlabs/manydepth#77]
I want to ask about the frozen network. I saw in the code that when you freeze the network, you put the frozen network in verification mode and use with torch.no_grad(): block gradient update, and detach() the result of the frozen network calculation when calculating the loss. I do not understand this also saw some information on the Internet is not very understand, there is no master to point out
